src/styles/LoginStyles.ts

```diff
@@ -1,29 +1,31 @@
 import styled from 'styled-components';
 import { Button, TextField, Typography, Link, Container } from '@material-ui/core';
 
-export const LoginContainer = styled(Container)`
-    margin-top: 4%;
+export const MainDiv = styled('div')`
+    margin-top: 80px;
+`;
+
+export const FieldDiv = styled('div')`
+
 `;
 
 export const Label = styled(Typography)`
     color: #828282;
     font-size: 12px;
     font-weight: bold;
-    text-align: left;
-    padding: 6px 0px;
-    margin-left: 8%;
-    display: block;
+    margin-top: -20px;
+    margin-left: -83%;
 `;
-  
+
 export const Field = styled(TextField)`
     background-color: #F8F8F8;
     outlined: false;
     border-radius: 15px;
-    width: 340px;
+    width: 95%;
     height: 69px;
```

Comment:
Using `height` results in the cursor/typing to be pushed to the top / not centered. The `padding` method that you had before might be able to replace this height line while keeping the cursor/typing centered in the TextField